src/test/librbd/operation/test_mock_SnapshotRemoveRequest.cc

```diff
@@ -148,7 +148,11 @@ TEST_F(TestMockOperationSnapshotRemoveRequest, Success) {
 
   C_SaferCond cond_ctx;
   MockSnapshotRemoveRequest *req = new MockSnapshotRemoveRequest(
-    mock_image_ctx, &cond_ctx, "snap1", snap_id);
```

Comment:
I'd rather keep the previous-style of indentation when you have the case where a full indent doesn't fit. This applies throughout.